package.json

```diff
@@ -142,7 +142,7 @@
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "64 kB"
+      "maxSize": "66 kB"
```

Comment:
is this still 66KB after the PRs I've made, or have you not yet rebased master?

Reply:
it was outdated https://github.com/algolia/instantsearch.js/pull/4464/commits/4c41441d628e30c130e66c71bd055b3824891d44